tests/ui_tests/test_ui_add_component_window.py

```diff
@@ -1660,6 +1660,24 @@ def test_UI_GIVEN_field_widget_with_stream_type_and_schema_set_to_f142_and_type_
     assert streams_widget.array_size_spinbox.isVisible()
 
 
+def test_UI_GIVEN_initial_component_THEN_webbrowser_url_is_correct(qtbot):
```

Comment:
Maybe `...webbrowser_url_contains_component_class`?

`...is_correct` doesn't really tell the reader anything useful.